docs/modules/ROOT/pages/getting-started/filters-and-projections.adoc

```diff
@@ -0,0 +1,191 @@
+
+You can define multiple columns to return in the projection by passing them into the `.return` method. This method not only accepts Nodes, but any kind of variables, such as properties or params.
+
+Let's update the return subclause:
+
+```javascript
+    .return(movieNode.property("title"), movieNode.property("tagline"), movieNode.property("year"));
+```
+
+The returned Cypher now should be:
+
+```cypher
+MATCH (this0:Movie)
+WHERE (this0.title = $param0 AND this0.released < $param1)
+RETURN this0.title, this0.tagline, this0.released
+```
+
+Success! You have now a query with filters and projections.
+
+=== Reusing variables
+
+Right now, this query is filtering and returning the properties `id` and `title` of the Movie nodes. However, the query relies on the name of these properties to remain the same. This, particularly with more complex queries, can lead to flaky queries, as it is hard to keep track of strings identifying values.
+
+To fix this, you'll make a small adjustment to the script, so that both the filters and the projection use the same property variable:
+
+```javascript
+const titleProp = movieNode.property("title");
+const yearProp = movieNode.property("released");
+const taglineProp = movieNode.property("tagline");
+
+const clause = new Cypher.Match(movieNode)
+    .where(Cypher.eq(titleProp, new Cypher.Param("The Matrix")))
+    .and(Cypher.lt(yearProp, new Cypher.Param(2000)))
+    .return(titleProp, taglineProp, yearProp);
+```
+
+This change, not only ensures that different parts of the query are always in sync but also makes the clause itself shorter and easier to read.
+
+[TIP]
+====
+Params can also be assigned to a variable and reused, this can be particularly useful when having multiple filters over the same parameter.
+====
+
+== Conclusion
+
+Your script should now look similar to this:
+
+```javascript
+import Cypher from "@neo4j/cypher-builder";
+
+const movieNode = new Cypher.Node({
+    labels: ["Movie"],
+});
+
+const titleProp = movieNode.property("title");
+const yearProp = movieNode.property("released");
+const taglineProp = movieNode.property("tagline");
+
+const clause = new Cypher.Match(movieNode)
+    .where(Cypher.eq(titleProp, new Cypher.Param("The Matrix")))
+    .and(Cypher.lt(yearProp, new Cypher.Param(2000)))
+    .return(titleProp, taglineProp, yearProp);
+
+const { cypher, params } = clause.build();
+console.log(cypher);
+console.log(params);
+```
+
+And its execution should show the following query:
+
+```cypher
+MATCH (this0:Movie)
+WHERE (this0.title = $param0 AND this0.released < $param1)
+RETURN this0.title, this0.tagline, this0.released
+```
+
+```javascript
+{ param0: 'The Matrix', param1: 2000 }
+```
+
+With this, you already have the tools to make useful (albeit simple) queries and have also learnt how to deal with parameters. Feel free to try other filters operations (TODO: add link )
+
+In the next tutorial, you'll add relationships and more complex filtering to this query.
```

Comment:
Same, I would add this in the beginning and then link to the other page about more complex filtering.